views/administration/forms/form-import.hbs

```diff
@@ -11,7 +11,7 @@
     <a href="https://docs.schul-cloud.org/x/NYBgAw" target="_blank">hier</a>.
 </p>
 <p>
-    <img src="/images/csv_example.png" />
+    <img src="/images/csv_example2.png" />
```

Comment:
why have you attached a `2` to the filename? Seems unnecessary to me.

Reply:
fixed.